tools/functional-tester/etcd-tester/checks.go

```diff
@@ -28,8 +29,32 @@ type hashChecker struct {
 	tt *tester
 }
 
+type leaseChecker struct {
+	leaseStressers []Stresser
+}
+
+type compositeChecker struct {
+	checkers []Checker
+}
+
+func newCompositeChecker(checkers []Checker) Checker {
+	return &compositeChecker{checkers}
+}
+
 func newHashChecker(tt *tester) Checker { return &hashChecker{tt} }
 
+func newLeaseChecker(leaseStressers []Stresser) Checker { return &leaseChecker{leaseStressers} }
+
+func (cchecker *compositeChecker) Check() error {
```

Comment:
please group the structs with their methods (e.g., `type leaseChecker struct; func newLeaseChecker(); func (*leaseChecker) Check() error`)